src/Algebra/Properties/CommutativeSemigroup.agda

```diff
@@ -0,0 +1,121 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some theory for commutative semigroup
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra using (Semigroup)
+open import Algebra.FunctionProperties using (Commutative)
+
+module Algebra.Properties.CommutativeSemigroup
+       {ℓ ℓ=} (H : Semigroup ℓ ℓ=) (open Semigroup H using (_≈_; _∙_))
```

Comment:
> I'm afraid it doesn't look like you've pushed these changes.

Sorry, I have missed something.
Now I change this to
```
open Semigroup H public hiding (_≈_; _∙_)
open EqR setoid
```
``hiding`` set in order to avoid clash with the import in the module decl.

> Please could you indent the module parameters by 2 spaces as per the style guide?
 I have taken this indentation from ``master``, for example,
Algebra.FunctionProperties.Consequences.Propositional.
      
Now I replace it with
```
module Algebra.Properties.CommutativeSemigroup
  {ℓ ℓ=} (H : Semigroup ℓ ℓ=) (open Semigroup H using (_≈_; _∙_))
```